compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -140,6 +140,16 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
   case IREE::Codegen::DispatchLoweringPassPipeline::
       CPUConvTileAndDecomposeExpert: {
     TilingConfig tilingConfig = getTilingConfigForPipeline(funcOp);
+    auto peelAttrName = StringAttr::get(funcOp.getContext(), "enable_peeling");
+    auto trueAttr = BoolAttr::get(funcOp.getContext(), true);
+
+    DictionaryAttr config = translationInfo.getConfiguration();
+    if (config) {
+      auto peelAttr = translationInfo.getConfiguration().getNamed(peelAttrName);
+      if (peelAttr && peelAttr->getValue() == trueAttr) {
+        pipelineOpts.enablePeeling = true;
+      }
+    }
```

Comment:
Can we create a function in CPUUtils, and reuse it here? It makes codes easier to read.